lib/realtime_web/controllers/tenant_controller.ex

```diff
@@ -10,22 +11,16 @@ defmodule RealtimeWeb.TenantController do
   alias Realtime.PostgresCdc
   alias Realtime.Tenants
   alias Realtime.Tenants.Cache
-  alias Realtime.Tenants.Connect
-
-  alias RealtimeWeb.Endpoint
+  alias RealtimeWeb.OpenApiSchemas.EmptyResponse
+  alias RealtimeWeb.OpenApiSchemas.ErrorResponse
+  alias RealtimeWeb.OpenApiSchemas.NotFoundResponse
+  alias RealtimeWeb.OpenApiSchemas.TenantHealthResponse
+  alias RealtimeWeb.OpenApiSchemas.TenantParams
+  alias RealtimeWeb.OpenApiSchemas.TenantResponse
+  alias RealtimeWeb.OpenApiSchemas.TenantResponseList
+  alias RealtimeWeb.OpenApiSchemas.UnauthorizedResponse
```

Comment:
Love it. I prefer explicit `alias`es over grouping them as it makes it easier to reorder or remove them.

Reply:
same! also quicker to ctrl+f for usage

Reply:
True, also, I suggest checking out Lexical as a LSP for Elixir. It is awesome.